.travis.yml

```diff
@@ -3,7 +3,11 @@ env:
   - GO111MODULE=on
 go:
   - 1.11.x
+before_script:
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.13.2
```

Comment:
The invocation of `sh` as `sh -s -- ...` is actually a bash-ism (and maybe other shells). `--` is not supported by posix sh, and `sh` may be posix sh depending on the setup of the build VM. I know this is what golangci lint says to do, so I'd keep it the same, but I'm commenting just in case you see some weirdness here.